controllers/grid/languages/LanguageGridCellProvider.inc.php

```diff
@@ -44,12 +44,12 @@ function getTemplateVarsFromRowColumn($row, $column) {
 			case 'uiLocale';
 				return array('selected' => $element['supportedLocales'],
 					'disabled' => !$element['supported']);
-			case 'submissionLocale';
```

Comment:
I don't think any of the changes to this file have any effect.

Reply:
(Maybe it's just to keep the code aligned with the order of presentation?)

Reply:
yes, just for that :-)